Automattic/sensei
# Add Upsell section for Premium Lesson Patterns

Fixes #5135

### Changes proposed in this Pull Request

* Adapt PatternsStep component to accept injection components between the title and the pattern list (6c9bf40dcbfb9507c35f6c5d58aaa6ffe9b29546);
* Add Upsell for the Premium Lesson Patterns (24a3c554f2cea481343b062d0d5c888136d06f00 and b200fb80b4071da9f6020741b426fa28a361c5ac)

### Testing instructions

On a WordPress installation with the code from this branch:

1. Create a new Lesson;
2. On the Editor Wizard, click "Continue" on the first step;
3. On the Lesson Patterns step, verify that the section "Want more lesson types, check out Sensei Pro." only appears if Sensei Pro is not installed;
4. Verify that the section "Want more lesson types, check out Sensei Pro." match the design (as per this post p6rkRX-3zM-p2);  
5. Verify that clicking the "Learn More" link opens the SenseiLMS.com pricing page in a new tab, and with some UTM tracking code as tracking parameters;

### Screenshot / Video

![Screenshot of the Lesson Patterns step with the Upsell section](https://user-images.githubusercontent.com/529864/172959303-13037c01-6911-42ca-905b-167f77e0b640.png)
